Quick band-aid fix for always visible 1-2 pixel bar

StefanOyx opened the pull request:
I've commented as best I can, -- xD is where I made changes. Hope you can figure out a much cleaner way to fix this after taking a closer look.

StefanOyx committed:
Quick band-aid fix for always visible 1-2 pixel bar

Zidras commented on ElvUI_ProjectZidras/Modules/UnitFrames/Elements/HealPrediction.lua:
```diff
@@ -7,7 +7,7 @@ local StatusBarPrototype = T.StatusBarPrototype
 function ZUF.HealthClipFrame_HealComm(frame)
 	if frame.HealCommBar then
-		ZUF:SetAlpha_HealComm(frame.HealCommBar, 1)
+		ZUF:SetAlpha_HealComm(frame.HealCommBar, 0) --xD changed from 1 to 0 to start it off hidden avoiding bug yet still have it there so it still positions properly
```
This breaks raid frame prediction, and I also find this not to be an adequate change since it affects bar creation

Zidras commented on ElvUI_ProjectZidras/Modules/UnitFrames/Elements/HealPrediction.lua:
```diff
@@ -303,6 +303,30 @@ function ZUF:UpdateHealComm(_, myIncomingHeal, otherIncomingHeal, absorb, _, has
+        absorbBar:SetAlpha(1)
+        overAbsorb:SetAlpha(1)
+    end
+
```
This would fail if you had one of the IncHeals present. Ideally, for the bandaid to work, I assume you'd have to check the Incoming value for 0 and AlphaHide the respective bar

Zidras commented on ElvUI_ProjectZidras/Modules/UnitFrames/Elements/HealPrediction.lua:
```diff
@@ -303,6 +303,30 @@ function ZUF:UpdateHealComm(_, myIncomingHeal, otherIncomingHeal, absorb, _, has
+    end
+
+	-- xD end of shit code
+
```
Same here, not all bars are absorb

Zidras committed:
Specify elements to be Alpha'd based on value == 0

Perhaps this method is a bit more robust than previous to avoid assumptions that are not always true
